### PR TITLE
Added check for -g and -gu for windows platform

### PR DESCRIPTION
**Pull request by hpohekar**
image

image

### CODE REVIEW

File: src/ansys/fluent/core/launcher/launcher.py [hunk 1]
```diff
@@ -302,13 +302,11 @@ def _update_launch_string_wrt_gui_options(
     if (show_gui is False) or (
         show_gui is None and (os.getenv("PYFLUENT_SHOW_SERVER_GUI") != "1")
     ):
-        if (
-            "-g " not in additional_arguments
-            and "-gu " not in additional_arguments
-            and not additional_arguments.endswith("-g")
-            and not additional_arguments.endswith("-gu")
-        ):
+        if not {"-g", "-gu"} & set(additional_arguments.split()):
```
Comment (prmukherj) [inline src/ansys/fluent/core/launcher/launcher.py:305]: @hpohekar, please look into the new location of the code and test it. Thank you.

**Review by prmukherj — APPROVED** [review]

**Review by mkundu1 — APPROVED** [review]

**Comment by seanpearsonuk** [comment]
@hpohekar Unfortunately e.g., `s=pf.launch_fluent(mode="solver", show_gui=True, additional_arguments="-g")` is still allowed and it starts in `-g` mode. The check needs to be more comprehensive.

**Review by seanpearsonuk — CHANGES_REQUESTED** [review]
As per previous comment. Perhaps we could also add a test for the expected behaviour.

File: src/ansys/fluent/core/launcher/launcher.py [hunk 2]
```diff
@@ -302,13 +302,11 @@ def _update_launch_string_wrt_gui_options(
     if (show_gui is False) or (
```
Comment (prmukherj) [inline src/ansys/fluent/core/launcher/launcher.py:302]: @seanpearsonuk, this was wrongly modified as  we had an assumption that -g and -gu is only used id show_gui=False. Thank you for clarifying it.

**Comment by hpohekar** [comment]
> @hpohekar Unfortunately, e.g., `s=pf.launch_fluent(mode="solver", show_gui=True, additional_arguments="-g")` is still allowed and it starts in `-g` mode. The check needs to be more comprehensive.

Done. Thank you.

**Comment by hpohekar** [comment]
> As per previous comment. Perhaps we could also add a test for the expected behaviour.

Done. Thank you.

File: src/ansys/fluent/core/launcher/launcher.py (hunk 2 above)
Comment (hpohekar) [reply src/ansys/fluent/core/launcher/launcher.py:308]: Right, Thank you.

**Comment by hpohekar** [comment]
> @hpohekar, please look at the current implementation. @seanpearsonuk, the tests should run locally without any issues, however though we have a case that should launch fluent in '-g' mode in linux as well. Let's see if the tests behave as we expect them to :).
> 
> >

Yes, looks good. Thank you.

**Review by mkundu1 — APPROVED** [review]

**Review by seanpearsonuk — APPROVED** [review]

**Pull request merged**